### DISCUSSION TITLE
tracee using wrong argument for attaching tracepoints

### DISCUSSION THREAD

**Discussion by Ankurk99** [aquasecurity/tracee#1146] [Development]
Issue in: tracee-ebpf
The AttachTracepoint() function (from libbpfgo) takes two arguments: event category and event name. But, currently tracee is providing only one argument i.e. probe.event to the function. 
This event string contains both the category and event name separated by a backslash '/' , which can be separated and provided as individual arguments to the AttachTracepoint() function.

**Comment by yanivagman** [comment]
Libbpfgo recently changed this API in https://github.com/aquasecurity/libbpfgo/pull/77
Tracee currently uses an older version of libbpfgo (before this change).
There is an open PR that updates libbpfgo and this function call as well: https://github.com/aquasecurity/tracee/pull/1125/files#diff-710a36e0ca7a07ea49cf4970d916ed0b74b1c3619920145d9db50477c4f96967R880

**Reply by Ankurk99** [reply]
Okay, got that. Thanks for the reply.
